working-parameters/README.md

```diff
@@ -0,0 +1,26 @@
+# Working Parameters for Private Measurement Technical Specification MVP
```

Comment:
Can we come up with a term other than "parameters" which invokes things like concrete epsilon levels, etc. Maybe "design dimensions" to align with the other doc?

Reply:
Sure, I'm fine with "design dimensions". I initially had "assumptions" written down, but that didn't feel quite right either.

Reply:
Is it accurate to refer to them as "constraints"? That's what I consider them and it may be more recognizable, domain-specific term for folks reviewing the doc.

Reply:
In some cases, the result is a constraint, but it seems like "design dimension" fits a bit better (and those dimensions may or may not be constrained in specific ways.) Does that make sense?